lib/web_ui/lib/src/engine/platform_dispatcher/app_lifecycle_state.dart

```diff
@@ -103,4 +110,12 @@ class _BrowserAppLifecycleState extends AppLifecycleState {
       onAppLifecycleStateChange(ui.AppLifecycleState.hidden);
     }
   });
+
+  void _onViewCountChanged(_) {
+    if (_viewManager.views.isEmpty) {
+      onAppLifecycleStateChange(ui.AppLifecycleState.detached);
+    } else {
+      onAppLifecycleStateChange(ui.AppLifecycleState.resumed);
```

Comment:
When there's only an implicit view, how do we transition to this `resumed` state?

Reply:
With the implicit view, we never go to detached.

Going to resumed will happen based on other events (just like before).